### ISSUE TITLE
Suggestion: HTTP status code should always return code 200

### ISSUE THREAD

**Issue by chiaki64** [ethe/pygraphy#26]
It seems that pygraphy always returning http status code 400 when error occurs, but i found other graphql frameworks mostly return 200 OK, like the Graphcool docs says 

```
HTTP status codes are not relevant when using GraphQL. 
A GraphQL server will always respond with code 200 (unless an internal server occured, then it might be code 500)
Any further information needs to be parsed from the JSON payload of the response.
```

And when i use apollo(graphQL client with js), if it return 400 status code, i can only get the Network error like `[Network error]: ServerError: Response not successful: Received status code 400`, and can't get GraphQL error, so the errors in the response is invisible to me.

So does pygraphy have plan to return an optional error status code(200 or 400)?

**Comment by ethe** [comment]
Yes, I think you get the right way, I will change this behavior later.

**Comment by ethe** [comment]
Fixed with #28
